editor/sidebar/post-visibility/index.js

```diff
@@ -100,9 +112,13 @@ class PostVisibility extends Component {
 						<div className="editor-post-visibility__dialog-legend">
 							{ __( 'Post Visibility' ) }
 						</div>
-						{ visibilityOptions.map( ( { value, label, info, changeHandler, checked } ) => (
+						{ visibilityOptions.map( ( { value, label, info, checked } ) => (
 							<label key={ value } className="editor-post-visibility__dialog-label">
-								<input type="radio" value={ value } onClick={ changeHandler } checked={ checked } />
+								<input
+									type="radio"
+									value={ value }
+									onChange={ this.setVisibility }
```

Comment:
It was initially a `onChange` and It just stopped working for a mysterious reason, that's why I switched to `onClick`